src/main/java/Chunky/Parser/Parser.java

```diff
@@ -0,0 +1,142 @@
+package Chunky.Parser;
+
+import Chunky.ChunkyException;
+import Chunky.InvalidMessageException;
+import Chunky.MissingArgumentException;
+import Chunky.Task.Deadlines;
+import Chunky.Task.Events;
+import Chunky.Task.Task;
+import Chunky.Task.ToDo;
+
+import java.util.ArrayList;
+
+/**
+ * Parses the input of the user and creates respective Task Objects
+ */
+public class Parser {
```

Comment:
Parser is generic. If this is the parser for tasks/commands, maybe CommandParser or TaskParser would better reflect domain specificity.

```suggestion
 */
public class CommandParser {
```